src/test/kotlin/uk/gov/justice/digital/hmpps/manageusersapi/resource/UserControllerIntTest.kt

```diff
@@ -673,27 +679,23 @@ class UserControllerIntTest : IntegrationTestBase() {
     @Test
     fun `User Me Roles endpoint returns principal user data for auth user`() {
       webTestClient
-        .get().uri("/roles/delius?deliusRoles=TEST_ROLE,TEST_WORKLOAD_MEASUREMENT_ROLE")
+        .get().uri("/roles/delius")
         .headers(setAuthorisation("AUTH_ADM", listOf()))
         .exchange()
         .expectStatus().isOk
-        .expectBody()
-        .jsonPath("[*].name").value<List<String>> {
-          assertThat(it).contains("ROLE_LICENCE_RO", "ROLE_GLOBAL_SEARCH", "ROLE_WORKLOAD_MEASUREMENT")
-        }
-    }
-
-    @Test
-    fun `User Me Roles endpoint returns principal user data for auth user1`() {
-      webTestClient
-        .get().uri("/roles/delius?deliusRoles=NON_MAPPED_ROLE")
-        .headers(setAuthorisation("AUTH_ADM", listOf()))
-        .exchange()
-        .expectStatus().isOk
-        .expectBody()
-        .jsonPath("[*].name").value<List<String>> {
-          assertThat(it).hasSize(0)
-        }
+        .expectBody().json(
+          """
+                           {
+                              "TEST_ROLE": [
+                                  "ROLE_LICENCE_RO",
+                                  "ROLE_GLOBAL_SEARCH"
+                              ],
+                              "TEST_WORKLOAD_MEASUREMENT_ROLE": [
+                                  "ROLE_WORKLOAD_MEASUREMENT"
+                              ]
+                          }
```

Comment:
I woiuld try to reformat this to make it more readable - 2 space indents (lol)

Reply:
done